build.gradle.kts

```diff
@@ -8,75 +7,80 @@ plugins {
   id("jacoco")
 }
 
-configurations {
-  testImplementation { exclude(group = "org.junit.vintage") }
-}
-
 dependencies {
-  annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
-  annotationProcessor("org.projectlombok:lombok:1.18.30")
-  testAnnotationProcessor("org.projectlombok:lombok:1.18.30")
-
-  implementation("org.springframework.boot:spring-boot-starter-aop")
-  implementation("org.springframework.boot:spring-boot-starter-validation")
-  implementation("org.springframework.boot:spring-boot-starter-jdbc")
+  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:2.1.1")
   implementation("org.springframework.boot:spring-boot-starter-data-jpa")
-  implementation("org.springframework.boot:spring-boot-starter-security")
-  implementation("org.springframework.boot:spring-boot-starter-cache")
-  implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:3.0.0")
-  implementation("io.hypersistence:hypersistence-utils-hibernate-63:3.7.5")
-  implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:2.4.0")
-  implementation("com.microsoft.azure:applicationinsights-logging-logback:2.6.4")
-
-  developmentOnly("org.springframework.boot:spring-boot-devtools")
-
+  implementation("org.springframework.boot:spring-boot-starter-hateoas")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+  implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.3.0")
+  implementation("com.fasterxml.jackson.core:jackson-databind")
+  implementation("io.jsonwebtoken:jjwt-impl:0.12.5")
```

Comment:
these are listed a `testImplementation` too?  do they needed to be listed only here instead or can you get away with them just being `testImplementation`?

Reply:
we need this here as well